docs/language/learn-ql/writing-queries/introduction-to-queries.rst

```diff
@@ -78,7 +78,6 @@ When writing your own alert queries, you would typically import the standard lib
 
 - C/C++: ``cpp``
 - C#: ``csharp``
-- COBOL: ``cobol``
 - Java: ``java``
```

Comment:
```suggestion
- Go: ``go``
- Java: ``java``
```

Reply:
Thanks for review. I had wondered about adding Go, but thought that we might have decided not to since it's not part of this repository. I couldn't remember. Happy to add if that's what we decided 😄

Reply:
Good question. I hadn't given it that much thought to be honest. But since we previously mentioned COBOL support, which was in a different private repo, I think it makes sense to add Go here. The Go tutorials will somehow be included in this Sphinx project when they arrive too. I think @shati-patel and @max-schaefer have been coordinating on that work.